README.md

```diff
@@ -1,19 +1,66 @@
 # odf-cli
 
-The ODF CLI tool provides configuration and troubleshooting commands for OpenShift Data Foundation. 
+The ODF CLI tool provides configuration and troubleshooting commands for OpenShift Data Foundation.
 
 ## Usage
 
-TBD
+`odf [root args] [command]`
+
+### Root args
+
+These are args currently supported:
+
+1. `-h|--help`: this will print brief command help text.
+
+    ```bash
+    odf -h
+    ```
+
+2. `-n|--namespace='openshift-storage'`: the Openshift namespace in which the StorageCluster resides. (optional,  default: openshift-storage)
+
+    ```bash
+    odf -n test-cluster [commands]
+    ```
+
+3. `-o|--operator-namespace` : the Openshift namespace in which the rook operator resides, when the arg `-n` is passed but `-o` is not then `-o` will equal to the `-n`. (default: openshift-storage)
+
+    ```bash
+    odf --operator-namespace test-operator -n test-cluster [commands]
+    ```
+
+4. `--context`: the name of the Openshift context to be used (optional).
+
+    ```bash
+    odf --context=$(oc config current-context) [commands]
+    ```
 
 ## Commands
 
-TBD
+- `set <args>`: Set the recovery profile to favor new IO, recovery, or balanced mode with options high_client_ops, high_recovery_ops, or balanced
```

Comment:
Is this intended to allow setting any ceph config value, or just the mclock profile? Is this more accurate to the implementation?

```suggestion
- `set recovery-profile <profile>`: Set the recovery profile to favor new IO, recovery, or balanced mode with options high_client_ops, high_recovery_ops, or balanced
```

Reply:
this is intended to allow only mclock profile config.